### PR TITLE
Test Script for Servos

### PR DESCRIPTION
**Pull request by mjliepke**
Pull request for the test script that will be used for the initial PWM test.  Could be a servo, esc or motor

### CODE REVIEW

File: SWDevelopment/TestScripts/ServoTestScript [hunk 1]
```diff
@@ -0,0 +1,28 @@
+  public static void main(String[] args)
+  {
+    PCA9685 servoBoard = new PCA9685();   // 0x40 is the default address
+    servoBoard.setPWMFreq(60); // Set frequency to 60 Hz
+    int servoMin = 150;   // Min pulse length out of 4096
+    int servoMax = 600;   // Max pulse length out of 4096
+
+    final int CONTINUOUS_SERVO_CHANNEL = 14;
```
Comment (mjliepke) [inline SWDevelopment/TestScripts/ServoTestScript:8]: Do we know how these channels layout on the PWM controller?  So when we run this test we know which one to plug it into?

Comment (kyleldoll) [reply SWDevelopment/TestScripts/ServoTestScript:8]: I would assume we have to change the CONTINUOUS_SERVO_CHANNEL value to whatever the channel is for the new controller, right? Also, isn't the servoMin & servoMax supposed to be a double between -1 and 1?

**Review by kyleldoll — APPROVED** [review]
The value of the channel may have to change, but the PWM could have the values listed in the comments; I'm used to the -1 to 1

File: SWDevelopment/TestScripts/ServoTestScript (hunk 1 above)
Comment (mjliepke) [reply SWDevelopment/TestScripts/ServoTestScript:8]: servoMin and servoMax are the actual PWM frequencies I assume, aka Hz for each 3.3 V -> 5V ->3.3 V transformation

**Pull request merged**